runtime/appruntime/exported/experiments/set.go

```diff
@@ -0,0 +1,127 @@
+package experiments
+
+import (
+	"os"
+	"strings"
+
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+
+	"encore.dev/appruntime/exported/config"
+)
+
+// Set is a set of experiments enabled within this app
+type Set struct {
+	enabled map[Name]struct{}
+}
+
+// NewForRuntime constructs a new Experiments object from both the static and runtime configs.
+//
+// It is used by the Encore runtime to construct the Experiments set in a running application.
+//
+// Unknown experiments are ignored.
+func NewForRuntime(static *config.Static, runtime *config.Runtime) *Set {
```

Comment:
It's a bit confusing that "ForRuntime" checks the compiled-in config, and "ForCompiler" checks the environment variable at runtime. It's not wrong but it's confusing.

Maybe rename this `FromConfig` and rename the other to `FromEnviron`?